policyengine_us/variables/gov/states/az/tax/income/deductions/standard/az_increased_standard_deduction.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class az_increased_standard_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona increased standard deduction"
+    unit = USD
+    documentation = "https://www.azleg.gov/viewdocument/?docName=https://www.azleg.gov/ars/43/01041.htm"
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.az.tax.income.deductions.standard
+        increased_percent = p.rate
+        az_charitable_contributions_credit = tax_unit(
+            "az_charitable_contributions_credit", period
```

Comment:
i think it is the charitable deduction and not the credit right?

Reply:
wouldn't it be then
        charitable_deduction_after_credit = max_(
            charitable_deduction - charitable_contributions_credit, 0
        )
and not just the credit?